docs/src/pages/components/HorizontalNav/WebHorizontalNav.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Horizontal navigation
+platform: web
+githubPath: bpk-component-horizontal-nav
+---
+
+import Readme from 'components/Readme';
+import PresentationBlock from 'components/PresentationBlock';
+import readme from 'bpk-component-horizontal-nav/README.md';
+import { 
+  DefaultExample, 
+  SpacedAroundExample, 
+  NotUnderlinedExample, 
+  LightAppearanceExample
+} from 'bpk-component-horizontal-nav/examples';
+import { colorSkyBlueShade02 } from 'bpk-tokens/tokens/base.es6';
+
+## Table of contents
+
+## Default
+
+By default, navigation items are left aligned.
+
+<PresentationBlock>
+  <DefaultExample />
+</PresentationBlock>
+
+## Space around
+
+Navigation items can be configured to have space around them.
+
+<PresentationBlock>
+  <SpacedAroundExample />
+</PresentationBlock>
+
+## Hide underline
+
+The border that covers the bottom of the component can be hidden.
+
+<PresentationBlock>
+  <NotUnderlinedExample />
+</PresentationBlock>
+
+## Light appearance
+
+An alternate appearance is available for display over dark backgrounds.
+
+<PresentationBlock style={{ backgroundColor: colorSkyBlueShade02 }}>
+  <LightAppearanceExample />
```

Comment:
We have `BpkDarkExampleWrapper` in the main repo for this, the idea is that the docs site should never have to be concerned about this presentation and everything is built into the example itself. 

In this case I'd recommend leaving the inline style in _for now_, but then in your next round of PRs, change `LightAppearanceExample` in `bpk-horizontal-nav/examples.js` to use `BpkDarkExampleWrapper`, then come back and remove the inline style from here as it will then not be needed.

Reply:
Yes the example is using this in the main repo but the container was not big enough for it to properly display which is why I have had to do this so that it could be shown off properly

Reply:
This is what it looks like without the inline style

<img width="703" alt="Screenshot 2021-03-12 at 10 47 35" src="https://user-images.githubusercontent.com/8831547/110929901-64e73c00-8320-11eb-9d44-a466f597e13c.png">

Reply:
Ah I see. I think we should still fix it in the web repo though, so this page can remain ignorant of style concerns. We could just add some bottom padding on the example in the web repo to make the issue go away

Reply:
`<PresentationBlock` has a prop for exactly this 🤔

Reply:
https://github.com/Skyscanner/backpack-docs/pull/925